google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -128,3 +128,5 @@ clouddebug.snapshots=Cloud Debugger Snapshots
 clouddebug.couldnotconnect=Could not connect to Google. Please make sure you have a working internet connection.
 clouddebug.stop.and.create.new.session=A cloud debugger session is already running.\nWould you like to stop it and start a new session?
 clouddebug.stop.background.and.create.new.session=A cloud debugger session is listening in the background.\nWould you like to stop it and start a new session?
+clouddebug.background.listener.error.title=Error while connecting to Cloud Debug backend
+clouddebug.background.listener.error.message=A cloud debug process is listening in the background but cannot access the backend, listening will stop now. Verify that there is a Google account signed in with access permissions to the project
```

Comment:
The idea for "listening will stop now" was to notify the user that the plugin will stop listening in the background for events from the backend. If we omit it, it might not be obvious to the user that this is not only a warning, but also something else have changed.